tests/lib/apt.sh

```diff
@@ -13,6 +13,28 @@ install_build_snapd(){
         mv sources.list.back /etc/apt/sources.list
         apt update
     else
-        distro_install_local_package "$GOHOME"/snapd_*.deb
+        packages=
+        case "$SPREAD_SYSTEM" in
```

Comment:
Idle wondering if the name for this file (`apt.sh`)  is still appropriate - maybe it should just get merged with pkgdb.sh - but that is a different PR I guess :)